File: app/src/main/java/com/improve10x/chatwithv2/base/BaseActivity.java
package com.improve10x.chatwithv2.base;

import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class BaseActivity extends AppCompatActivity {

    protected void showToast(String template) {
        Toast.makeText(this, template, Toast.LENGTH_SHORT).show();
    }

}

File: app/src/main/java/com/improve10x/chatwithv2/base/BaseActivity.kt
package com.improve10x.chatwithv2.base

import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

open class BaseActivity : AppCompatActivity() {

    protected fun showToast(message: String) {
        Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
    }
}